testnet/testnet-deploy-l2-contracts.sh

```diff
@@ -56,21 +66,39 @@ do
 done
 
 # ensure required fields
-if [[ -z ${l2host:-} || -z ${deployer_pkstring:-} || -z ${hocpkstring:-} || -z ${pocpkstring:-} ]];
+if [[ -z ${l2host:-} || -z ${deployer_pkstring:-} || -z ${hocpkstring:-} || -z ${pocpkstring:-} ||  -z ${l1host:-} ]];
 then
     help_and_exit
 fi
 
 # deploy contracts to the obscuro network
 echo "Deploying Token ERC20 contract to the obscuro network..."
 
-network_cfg='{ 
+network_cfg='{
```

Comment:
Can there be a comment stating where the network cfg flags can be read / described ?